tony-core/src/main/resources/tony-default.xml

```diff
@@ -145,6 +145,13 @@
     <value>1</value>
   </property>
 
+  <!-- Container configurations -->
+  <property>
+    <description>Timeout, in milliseconds for allocated container registration before forcibly killing application.</description>
+    <name>tony.container.timeout</name>
```

Comment:
rename to "tony.container.allocation.timeout"

Reply:
Code updated. @oliverhu Please review again. Thanks.